v2/tools/generator/internal/interfaces/kubernetes_resource_interface.go

```diff
@@ -384,6 +389,62 @@ func newGetResourceScopeFunction(r *astmodel.ResourceType) func(k *functions.Obj
 	}
 }
 
+// newGetSupportedOperationsFunction creates a function that returns the supported operations of the resource.
+//
+//	func (<receiver> *<receiver>) GetSupportedOperations() []genruntime.ResourceOperation {
+//		return []genruntime.ResourceOperation{
+//			genruntime.ResourceOperationGet,
+//			genruntime.ResourceOperationPut,
+//			genruntime.ResourceOperationDelete,
+//		}
+//	}
+func newGetSupportedOperationsFunction(r *astmodel.ResourceType) func(k *functions.ObjectFunction, codeGenerationContext *astmodel.CodeGenerationContext, receiver astmodel.TypeName, methodName string) *dst.FuncDecl {
```

Comment:
Use the handler type for clarity

```suggestion
func newGetSupportedOperationsFunction(r *astmodel.ResourceType) astmodel.ObjectFunctionHandler {
```

Reply:
Doesn't apply anymore because I changed it to the resourceFunction signature.